docs/Advanced-Topics-Block-Components.md

```diff
@@ -73,9 +73,24 @@ It is strongly recommended that you use `editable: false` if your custom
 component will not contain text.
 
 If your component contains text as provided by your `ContentState`, your custom
-component should compose a `DraftEditorBlock` component. This will allow the
+component should compose a `EditorBlock` component. This will allow the
 Draft framework to properly maintain cursor behavior within your contents.
 
+By default it must have props defined. You can set all properties on the `EditorBlock` to those passed in by a `blockRendererFn` as follows:
```

Comment:
I think the sentence `By default it must have props defined` is a little confusing. How about:

"You should apply `this.props` to your `EditorBlock` to pass along props supplied by `blockRendererFn`."